tox.ini

```diff
@@ -4,17 +4,18 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py{27,35}-django{18,110,111}
+envlist = py{27,36}-django{18,110,111}
 
 [testenv]
 passenv = CI TRAVIS TRAVIS_*
 deps =
     django18: Django>=1.8,<1.9
     django110: Django>=1.10,<1.11
     django111: Django>=1.11,<2.0
-    -r requirements/test.txt
+    -rrequirements/test.txt
```

Comment:
The space is allowed as of tox 3.0.0, and better matches typical pip command line usage.

Reply:
Ah - I wondered about that!